input/examples/organization-kioma-pathology.xml

```diff
@@ -0,0 +1,53 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<Organization xmlns="http://hl7.org/fhir">
+    <id value="kioma-pathology"/>
+    <extension url="http://hl7.org/fhir/StructureDefinition/resource-instance-name">
+        <valueString value="Organization - Kioma Pathology Laboratory"/>
+    </extension>
+    <extension url="http://hl7.org/fhir/StructureDefinition/resource-instance-description">
+        <valueMarkdown value="Shows an example of Kioma Pathology Laboratory organisation for the *AU Core Organization* profile, including its HPI-O, type as a pathology laboratory, contact information and physical address."/>
+    </extension>
+    <meta>
+        <profile value="http://hl7.org.au/fhir/core/StructureDefinition/au-core-organization"/>
+    </meta>
+    <identifier>
+        <type>
+            <coding>
+                <system value="http://terminology.hl7.org.au/CodeSystem/v2-0203"/>
+                <code value="NOI"/>
+                <display value="National Organisation Identifier"/>
+            </coding>
+            <text value="HPI-O"/>
+        </type>
+        <system value="http://ns.electronichealth.net.au/id/hi/hpio/1.0"/>
+        <value value="8003621566705995"/>
+    </identifier>
+    <type>
+        <coding>
+            <system value="http://www.abs.gov.au/ausstats/abs@.nsf/mf/1292.0"/>
+            <code value="8520"/>
+            <display value="Pathology and Diagnostic Imaging Services"/>
+        </coding>
+        <text value="Pathology laboratory"/>
+    </type>
+    <name value="Kioma Pathology"/>
+    <telecom>
+        <system value="email"/>
+        <value value="info@kiomapathology.example.net"/>
+        <use value="work"/>
+    </telecom>
+    <telecom>
+        <system value="phone"/>
+        <value value="(07) 555 02887"/>
+        <use value="work"/>
+    </telecom>
+    <address>
+        <use value="work"/>
+        <type value="physical"/>
+        <line value="15 Hazlett Est"/>
+        <city value="Kioma"/>
+        <state value="QLD"/>
+        <postalCode value="44498"/>
```

Comment:
Correct postcode to 4498 as per test data

Reply:
fixed